### PR TITLE
mining voucher

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
half-ports mining voucher from tg
currently it has 3 of 6 kits:
- crusher kit, self explanatory
- extraction kit, fulton beacon and 20 fultons
- conscription kit

you press Z, select one, then use it on the salvage vendor to redeem it and get your goodies

Requires #2510 (conscription kit)

## Why / Balance
lets miners pick different classes at the start of the shift, like a crusher for faster mining or fultons for roundstart debris farming

## Technical details
uses a radial menu

## Media
...

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
no

**Changelog**
:cl:
- add: Added mining vouchers for salvage to get a loadout that can be redeemed at the salvage vendor.

### CODE REVIEW

**Comment by Lyndomen** [comment]
Shit I forgor to review, 12 hours

**Comment by Lyndomen** [comment]
I'm happy to ask dumb questions, why not use loadouts

**Comment by deltanedas** [comment]
you cant talk to eachother in the locker room and discuss what gear to use vs loadouts everyone took conscription billions must mine

**Comment by deltanedas** [comment]
> 12 hours

> 18 hours ago

erm what the gleep

**Comment by Lyndomen** [comment]
> > 12 hours
> 
> > 18 hours ago
> 
> erm what the gleep

erm i was uhhhh not doing ss14 stuff

File: Resources/Locale/en-US/deltav/salvage/mining-voucher.ftl [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+mining-voucher-minebot-name = Minebot Kit
+mining-voucher-minebot-description =
+    Contains a little minebot companion that helps you in storing ore and hunting wildlife.
+    Also comes with an upgraded industrial welding tool, a welding mask and a KA modkit that allows shots to pass through the minebot.
+
+mining-voucher-conscription-name = Mining Conscription Kit
+mining-voucher-conscription-description =
+    Contains a whole new mining starter kit for one crewmember, including
+    a proto-kinetic accelerator, mineral scanner, ID card and other useful gear.
```
Comment (Lyndomen) [inline Resources/Locale/en-US/deltav/salvage/mining-voucher.ftl]: no id pls  (ask LO to do their job)

Comment (Lyndomen) [reply Resources/Locale/en-US/deltav/salvage/mining-voucher.ftl]: damn that kit isnt in this pr

**Review by Lyndomen — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/deltav/salvage/mining-voucher.ftl (hunk 1 above)
Comment (deltanedas) [reply Resources/Locale/en-US/deltav/salvage/mining-voucher.ftl]: ok real ill remove it

**Review by Lyndomen — APPROVED** [review]
nice

**Pull request merged**